src/gwc/src/main/java/org/geoserver/gwc/config/GeoserverXMLResourceProvider.java

```diff
@@ -23,8 +26,12 @@
 
 public class GeoserverXMLResourceProvider implements ConfigurationResourceProvider {
 
-    private static Log log = LogFactory.getLog(org.geowebcache.config.XMLFileResourceProvider.class);
-    
+    private static Log LOGGER = LogFactory.getLog(org.geowebcache.config.XMLFileResourceProvider.class);
+
+    public static final String GEOWEBCACHE_CONFIG_DIR_PROPERTY = "GEOWEBCACHE_CONFIG_DIR";
+
+    public static final String GEOWEBCACHE_CACHE_DIR_PROPERTY = "GEOWEBCACHE_CACHE_DIR";
+
     public static final String DEFAULT_CONFIGURATION_DIR_NAME = "gwc";
```

Comment:
Waiting for the change on ``XMLFileResourceProvider.GWC_CONFIG_DIR_VAR`` to be merged.